examples/model_service_template/model_handler.py

```diff
@@ -73,6 +73,7 @@ def handle(self, data, context):
         :param context: model server context
         :return: list of outputs to be send back to client
         """
+        self.error = None  # reset earlier errors
```

Comment:
I am wondering the use of these template files now. We have had a few folks in the MMS user community who were confused why there were these many classes and inheritances. Should we just revamp this code and have all of this in one file? I was planning on replacing all these files with a single service file.

Reply:
Hmm. I re-read the comment. I think I will raise a new PR with revamping the model-files. I think we should have this fix in until then. Just curious, is changing all the model-zoo models, which use this file, also in the scope for this PR? Or should we take a separate task to do that?